src/parse.c

```diff
@@ -853,10 +853,15 @@ static void parse_import(const Token **tokens, const char *stdlib_path, Toplevel
     ++*tokens;
 
     bool parens = is_operator(*tokens, "(");
-    if(parens) ++*tokens;
+    if(parens) ++*tokens;
+    /*
+    if (!strcmp((*tokens)->data.name, '*') && (*tokens)->type != TOKEN_NAME)
+        printf("%d", !strcmp((*tokens)->data.name, '*'));
+        printf("\n");
+    */
 
     do {
-        if ((*tokens)->type != TOKEN_NAME)
+        if ((*tokens)->type != TOKEN_NAME && (*tokens)->type != TOKEN_OPERATOR) // Hah, failed many times make it to check *(tokens)->data.name == '*' successed
```

Comment:
You can use the `is_operator()` function:
```suggestion
        if ((*tokens)->type != TOKEN_NAME && !is_operator(*tokens, "*"))
```